students/biegon_piotr/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,22 @@
+print("Knight move\n")
+
+print("Enter the start coordinates:")
+x1 = int(input("x1 = "))
```

Comment:
it's also possible to take input in pairs, like:
`x1, y1 = int(input()), int(input())`